magma_access_gateway_installer/agw_preinstall_checks.py

```diff
@@ -54,11 +44,14 @@ def preinstall_checks(self):
         else:
             logger.info("Magma AGW pre-install checks completed. Starting installation...")
```

Comment:
`Starting installation...` should come from the method that actually starts the installation

Reply:
Removed.